.github/workflows/main.yml

```diff
@@ -267,3 +267,41 @@ jobs:
         path: ./simple_use_case/aiida/paper.pdf
         retention-days: 1
         if-no-files-found: error
+
+  run-gwl:
+    # guix uses sandboxing features which are not available in the container
```

Comment:
Not sure that I understand that comment, is there maybe additional links available that shed some light on these sandboxing features you are talking about?

Reply:
Yes, the guix-daemon requires at least `chroot` capabilities (see https://guix.gnu.org/en/manual/en/html_node/Build-Environment-Setup.html). This isn't supported in non-`--privileged` docker containers as run by github default runners, as far as I know.

Reply:
I've added some context to the comment.